examples/serialize_to_disk.rs

```diff
@@ -0,0 +1,186 @@
+//! This example demonstrates how to save [hecs::World] instance
+//! to disk and load it back to memory using serialization. It can be useful for implementing
+//! a save mechanism for a game.
+//!
+//! The example creates a sample `World`, serializes it, saves it to disk as `saved_world.world` file,
+//! loads it back from the disk, deserializes the loaded world data and validates that component
+//! data of the worlds match.
+//!
+//! Run this example from crate root with:
+//! `cargo run --example serialize_to_disk --features "column-serialize"`
+
+#[cfg(feature = "column-serialize")]
```

Comment:
I'm not sure what the point of this cfg gate is, because this will fail to compile regardless if the feature is disabled. Instead, define the feature as required for the example in Cargo.toml.

Reply:
True!

Reply:
This comment has not been addressed.

Reply:
Is it fine now?